src/worker/storage.ts

```diff
@@ -111,12 +58,25 @@ export class PredictiveTextStudioDexie extends Dexie {
        */
       projectData:
         "++id, langName, bcp47Tag, authorName, modelID, copyright, version",
+      /**
+       * KMP keyboardData Table Scehma
+       * +------------------+
+       * | id (primary key) |
+       * +------------------+
+       * +------------------+
+       * | bcp47Tag         |
+       * +------------------+
+       * | language         |
+       * +------------------+
+       */
+      keyboardData: "++id, bcp47Tag, langauge",
```

Comment:
I have a degree in linguistics and my MSc was in language models. I still can't spell "language" right on my first try 💀  — lol so please correct this:

```suggestion
      keyboardData: "++id, bcp47Tag, language",
```